packages/vscode-tailwindcss/src/extension.ts

```diff
@@ -684,24 +623,20 @@ export async function activate(context: ExtensionContext) {
   context.subscriptions.push(Workspace.onDidOpenTextDocument(didOpenTextDocument))
   Workspace.textDocuments.forEach(didOpenTextDocument)
   context.subscriptions.push(
-    Workspace.onDidChangeWorkspaceFolders((event) => {
-      for (let folder of event.removed) {
-        let client = clients.get(folder.uri.toString())
-        if (client) {
-          searchedFolders.delete(folder.uri.toString())
-          clients.delete(folder.uri.toString())
-          client.stop()
-        }
-      }
+    Workspace.onDidChangeWorkspaceFolders(async () => {
+      if (Workspace.workspaceFolders?.length ?? 0 > 0) return
```

Comment:
I think you want `()` here because of operator precedence, it just happens to work because numbers are truthy _and_ we don't have negative lengths _and_ `0` is falsey.

```suggestion
      if ((Workspace.workspaceFolders?.length ?? 0) > 0) return
```

Because right now it's doing `expression ?? (0 > 0)`:
```
> 20 ?? 0 > 0
20
> 20 ?? (0 > 0)
20
> (20 ?? 0) > 0
true
```

Reply:
Fixed. I pulled it out to a separate variable. Feels cleaner.